docs/INSTALLATION.md

```diff
@@ -0,0 +1,78 @@
+## Installation of Sulong
```

Comment:
I think we can remove the paragraph under this headline, since the `README.md` already covers the basic installation procedure.